driver/src/orderbook/streamed/updating_orderbook.rs

```diff
@@ -64,6 +80,14 @@ impl UpdatingOrderbook {
             .past_events(BlockNumber::Number(from_block.into()), to_block)
             .await?;
         self.handle_events(context, events, from_block).await?;
+        if self.filestore.is_some() {
```

Comment:
nit: to avoid the unwrap below you can do

`if let Some(filestore) = self.filestore { ...`